### ISSUE TITLE
Converting a raw image to jpeg using rawler

### ISSUE THREAD

**Issue by mukeshsoni** [dnglab/dnglab#499]
How can i convert a raw image to a jpeg using rawler? Are there any example code snippets somewhere. Some sample code to do that would be really helpful. Thanks.

**Comment by mukeshsoni** [comment]
@CyberTimon It does save to a png file but the png file is huge. 3 times the original RAW image itself.

And when i try to save as jpeg (with `rotated_image.save_with_format(output_path, image::ImageFormat::Jpeg)?`), i get this error 

```
Error: The encoder or decoder for Jpeg does not support the color type `Rgb16`
```

Thanks for your help!

**Comment by CyberTimon** [comment]
Yes that's correct. We don't change the underlying develop function. What's new is that we compress the image more aggressively, using the `::new_with_quality function` from jpegencoder:

```JpegEncoder::new_with_quality(&mut output_file, 85);```

The `, 85` part says how much quality we want in the final jpeg (less = more blocky artefacts = smaller image, 100 = original quality).

**Comment by mukeshsoni** [comment]
Unfortunately, the develop pipeline inside rawler is producing images which look quite different to the embedded jpeg inside the raw images for Fuji raw files :(. Some look faded, some have a purple tint, some have a green tint.
